ale/drivers/cube_driver.py

```diff
@@ -0,0 +1,224 @@
+        results['AngularVelocities'] = np.array( [ [av1, av2, av3] for av1, av2, av3 in zip(field_data['AV1'],field_data['AV2'],field_data['AV3']) ] )
+    if 'ET' in field_data:
+        results['Times'] = np.array(field_data['ET'])
+    if all (key in field_data for key in ('J2000Ang1','J2000Ang2','J2000Ang3')):
+        results['EulerCoefficients'] = np.array([field_data['J2000Ang1'],field_data['J2000Ang2'],field_data['J2000Ang3']])
+        results['BaseTime'] = field_data['J2000Ang1'][-1]
+        results['TimeScale'] = field_data['J2000Ang2'][-1]
+
+    if 'TimeDependentFrames' in label:
+        results['TimeDependentFrames'] = np.array(label['TimeDependentFrames'])
+    if all (key in label for key in ('ConstantRotation','ConstantFrames')):
+        const_rotation_mat = np.array(label['ConstantRotation'])
+        results['ConstantRotation'] = quaternion.from_rotation_matrix(np.reshape(const_rotation_mat, (3, 3)))
+        results['ConstantFrames'] = np.array(label['ConstantFrames'])
+    if all (key in label for key in ('PoleRa','PoleDec','PrimeMeridian')):
+        results['BodyRotationCoefficients'] = np.array( [label['PoleRa'],label['PoleDec'],label['PrimeMeridian']] )
+    if all (key in label for key in ('PoleRaNutPrec','PoleDecNutPrec','PmNutPrec','SysNutPrec0','SysNutPrec1')):
+        results['SatelliteNutationPrecessionCoefficients'] = np.array( [label['PoleRaNutPrec'],label['PoleDecNutPrec'],label['PmNutPrec']] )
+        results['PlanetNutationPrecessionAngleCoefficients'] = np.array( [label['SysNutPrec0'],label['SysNutPrec1']] )
+    return results
+
+def parse_position_table(field_data):
+    results = {}
+    if all (key in field_data for key in ('J2000X','J2000Y','J2000Z')):
+        results['Positions'] = np.array( [ [x, y, z] for x, y, z in zip(field_data['J2000X'],field_data['J2000Y'],field_data['J2000Z']) ] )
+    if 'ET' in field_data:
+        results['Times'] = np.array(field_data['ET'])
+    if all (key in field_data for key in ('J2000XV','J2000YV','J2000ZV')):
+        results['Velocities'] = np.array( [ [x, y, z] for x, y, z in zip(field_data['J2000XV'],field_data['J2000YV'],field_data['J2000ZV']) ] )
+    if all (key in field_data for key in ('J2000SVX','J2000SVY','J2000SVZ')):
+        results['PositionCoefficients'] = np.array( [field_data['J2000SVX'][:-1],field_data['J2000SVY'][:-1],field_data['J2000SVZ'][:-1]] )
+        results['BaseTime'] = field_data['J2000SVX'][-1]
+        results['TimeScale'] = field_data['J2000SVY'][-1]
+    return results
+
+class Cube(Base):
+
+    def __init__(self, file, *args, **kwargs):
+        super(Cube, self).__init__('')
+        self.label = pvl.load(file)
+        for table in self.label.getlist('Table'):
+            binary_data = read_table_data(table, file)
+            field_data = parse_table_data(table, binary_data)
+            if table['Name'] == 'InstrumentPointing':
+                self.inst_pointing_table = parse_rotation_table(table, field_data)
```

Comment:
If these tables vars are only used internally, probably best to prefix with an underscore like in other drivers.